doc/api/process.md

```diff
@@ -36,6 +36,31 @@ termination, such as calling [`process.exit()`][] or uncaught exceptions.
 The `'beforeExit'` should *not* be used as an alternative to the `'exit'` event
 unless the intention is to schedule additional work.
 
+```js
+process.on('beforeExit', (code) => {
+  console.log('Process beforeExit event with code: ', code);
+});
+
+process.on('exit', (code) => {
+  console.log('Process exit event with code: ', code);
+});
+
+(function doSomething() {
+  console.log('This message is displayed first.');
+})();
+
+// Prints:
+// doSomething: This message is displayed first.
+// beforeExit: Process beforeExit event with code: 0
+// exit: Process exit event with code: 0
+// Considerations:
+// If exit event is called then beforeExit event will not be executed
+// (function doSomething() {
+//   console.log('This message is displayed first.');
+//   process.exit();
+// })();
```

Comment:
This part is already described in [line 33](https://github.com/nodejs/node/pull/28430/files#diff-eaa7a086b3a1bc26c044f11e69800239R33). I personally would just remove the whole comment.